src/components/Book.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
```

Comment:
According to pathwright `book.js` should not connect to `redux store`, the screenshot below;
Kindly follow the instructions as other students are also following it 👍 

![image](https://user-images.githubusercontent.com/31897434/82297521-acc05f80-99aa-11ea-8185-f5d79f9a6c99.png)

Reply:
Solved!

`eraseBook` function comes from `BooksList` using props drilling:

https://github.com/mcastorena0316/Bookstore_CMS/blob/b833bc1e94a0d626112aa06353e5f49fefbbf10b/src/components/Book.js#L11-L14

... so is not connected to `redux` anymore:

https://github.com/mcastorena0316/Bookstore_CMS/blob/b833bc1e94a0d626112aa06353e5f49fefbbf10b/src/components/Book.js#L50